map_gen/maps/crash_site/commands.lua

```diff
@@ -42,6 +42,10 @@ callback =
             global_data.restarting = nil
             return
         elseif state == 1 then
+            local game_seconds = math.floor(game.ticks_played / 60) -- should probably account for game speed changes, but we rarely (if ever?) change crash site performance scale. This is good enough?
+            local game_hours = math.floor(game_seconds / 3600)
+            local game_min_remainder = math.floor((game_seconds - (game_hours * 3600)) / 60)
+            Server.to_discord_raw('<@&762441731194748958> Crash Site has just restarted!! Previous map lasted: ' .. game_hours .. ' hours, ' .. game_min_remainder .. ' minutes')
```

Comment:
Add markdown to make it **bold**?

```suggestion
            Server.to_discord_raw('<@&762441731194748958> **Crash Site has just restarted!! Previous map lasted: ' .. game_hours .. ' hours, ' .. game_min_remainder .. ' minutes**')
```

Reply:
It might also be woth having a local variable for `local crashsite_role ='<@&762441731194748958>'` just to explain what that number is.

Reply:
All good suggestions, thank you. I'll make the changes and test tonight.

Reply:
I think I'll also try work out how to put the notification in #map-update not #s7 so that all the map pings stay in one place and people don't have to scroll through the server channel.